projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php

```diff
@@ -91,6 +91,29 @@ public static function init() {
 		do_action( 'jetpack_mu_wpcom_initialized' );
 	}
 
+	/**
+	 * Schedules translation updates for Jetpack MU WPCOM.
+	 *
+	 * This function sets up the necessary cron jobs to ensure that translation files
+	 * are regularly updated.
+	 *
+	 * @return void
+	 */
+	public static function schedule_translation_updates() {
+		// Only schedule the cron job if it hasn't been scheduled before.
```

Comment:
Is the intention to allow the site admin to unschedule the event if they don't want translation updates to ever happen?

Reply:
In that case, doesn't the use of `if ( ! wp_next_scheduled( 'wpcomsh_daily_translation_update' ) )` already handle it? Why the option as well?